blockchain/chain/src/store/chain_store.rs

```diff
@@ -570,7 +570,15 @@ where
 
                 // Don't include identity CIDs.
                 if u64::from(Code::Identity) != cid.hash().code() {
-                    tx_clone.send_async((cid, block.clone())).await?;
+                    // We only include raw and dagcbor, for now.
```

Comment:
It'd be more concise with a single `if` condition e.g., `if code != identity && (codec == IPLD_RAW || codec == DAG_CBOR) { send_async } `.

Reply:
Good suggestion. `cargo fmt` kind of mangled the formatting of the match statement, so this way definitely looks cleaner.